modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/service/impl/SXPElementLocalServiceImpl.java

```diff
@@ -45,8 +45,8 @@ public class SXPElementLocalServiceImpl extends SXPElementLocalServiceBaseImpl {
 	public SXPElement addSXPElement(
 			String externalReferenceCode, long userId,
 			Map<Locale, String> descriptionMap, String elementDefinitionJSON,
-			boolean readOnly, String schemaVersion,
-			Map<Locale, String> titleMap, int type,
+			String fallBackDescription, String fallBackTitle, boolean readOnly,
```

Comment:
Is is really needed to pass this values around and to the service? Could those just be picked from the localization maps, because we have the current locale value there all the time?

Reply:
not just import scenario but copy scenario too